modules/local/print_contamination_angsd.nf

```diff
@@ -1,5 +1,5 @@
 process PRINT_CONTAMINATION_ANGSD {
-    tag "$meta.id"
+    tag "nuclear_contamination.txt"
```

Comment:
Could also be removed. The provided tag does not really provide the user with any information. The job should then be shown without a `()` next to the job name i believe.

Reply:
I think it would be better to not have empty brackets, just so it doesn't look like a bug, but am happy to change the content.

Reply:
I agree that empty brackets might be more confusing, but I believe that if no `tag` is given at all it does not print any brackets at all. 🤔 
Feel free to leave it as is if I'm wrong